t/io/argv.t

```diff
@@ -129,7 +129,9 @@ open STDIN, 'tmpIo_argv1.tmp' or die $!;
 @ARGV = ();
 ok( !eof(),     'STDIN has something' );
 
-is( <>, "ok 11\n" );
+my $stdin = <>;
+my $expect = "ok 11\n";
+is( $stdin, $expect, "Got what was expected from STDIN" );
```

Comment:
adding test description is great!

Reply:
Why not simply
```
is (scalar <>, "ok 11\n", "Got what was expected from STDIN");
```
instead of creating two new variables that are not used anywhere else?

Reply:
I had to stare at this for several minutes before I realized that `"ok 11\n"` was not the test description but the expectation.  Assigning to variables makes what was happening more self-documenting and will help the next maintainer.